lua/telescope/builtin.lua

```diff
@@ -45,14 +46,18 @@ builtin.git_files = function(opts)
     opts.cwd = vim.fn.expand(opts.cwd)
   end
 
+  opts.track_results = vim.g["telescope_cache_results"]
+  local use_prime = vim.g["telescope_prime_fuzzy_find"]
+
   pickers.new(opts, {
     prompt    = 'Git File',
     finder    = finders.new_oneshot_job(
       { "git", "ls-files", "-o", "--exclude-standard", "-c" },
       opts
     ),
     previewer = previewers.cat.new(opts),
-    sorter    = sorters.get_fuzzy_file(),
+    sorter    = use_prime and sorters.get_prime_fuzzy_file(opts) or
```

Comment:
If we add the configuration, it should apply to all file based telescope builtins.

I'm not sure I want to 100%. People can just call the builtins with `sorter = sorters.get_prime_fuzzy_file()` or similar.